Test/Pester5.Tests.ps1

```diff
@@ -488,15 +328,24 @@ Describe 'FailingTests'{
 }
 '@
         WhenPesterTaskIsInvoked -AsJob -WithArgument @{
-            # Make sure the Pester5 task's default values for these get overwritten
-            'Output' = '.output\pester.xml';
-            'OutputFormat' = 'JUnitXml';
-            # Make sure these do *not* get overwritten.
-            'PassThru' = $false;
-            # Make sure this gets passed.
-            'TestName' = 'PassingTests';
+            Debug = @{
+                ShowFullErrors = ($DebugPreference -eq 'Continue');
+                WriteDebugMessages = ($DebugPreference -eq 'Continue');
+            };
+            Run = @{
+                Path = 'PassingTests.ps1';
+                PassThru = $false;
+            };
+            Should = @{
+                ErrorAction = $ErrorActionPreference;
+            };
+            TestResult = @{
+                Enabled = $true;
+                OutputPath = '.output\pester.xml';
+                OutputFormat = 'JUnitXml';
+            };
         }
-        ThenPesterShouldHaveRun -FailureCount 0 -PassingCount 1 -AsJUnitXml -ResultFileName 'pester.xml'
+        ThenDidNotFail -AsJUnitXml -ResultFileName 'pester.xml'

```

Comment:
Here you want to also assert that the test results were published with `Publish-WhiskeyPesterTestResult`.